doc/users/whats_new/reproducible_ps_pdf.rst

```diff
@@ -0,0 +1,6 @@
+Reproducible PS and PDF output
+------------------------------
+
+The ``SOURCE_DATE_EPOCH`` environment variable can now be used to set
+the timestamps value in the PS and PDF outputs, which are then
+reproducible.
```

Comment:
Please add a link to the specification that describes the value of this environment variable.